### PR TITLE
Log the title text in the graphical version

### PR DESCRIPTION
**Pull request by chris-addison**

### CODE REVIEW

File: Assets/tiny_rogue/Scenes/MainScene_sprites.scene [hunk 1]
```diff
@@ -1034,9 +1034,6 @@
         }, {
             "$type": "Unity.Tiny.Core2D.Parent, Unity.Tiny.Core2D",
             "Value": "5aa6b81e39954e839ccd7dc875d06315"
-        }, {
```
Comment (mdiluz) [inline Assets/tiny_rogue/Scenes/MainScene_sprites.scene]: Not sure why there's so many removals

File: Assets/tiny_rogue/Scenes/MainScene_sprites.scene [hunk 2]
```diff
@@ -1886,8 +1880,7 @@
             "$type": "game.DungeonGenerator, tiny_rogue",
             "MaxNumberOfRooms": 11,
             "MinRoomSize": 6,
-            "MaxRoomSize": 10,
-            "NumberOfCollectibles": 5
+            "MaxRoomSize": 10
```
Comment (mdiluz) [inline Assets/tiny_rogue/Scenes/MainScene_sprites.scene]: This change seems unrelated

**Review by mdiluz — CHANGES_REQUESTED** [review]

**Review by ObsessiveNerd — APPROVED** [review]

**Review by mdiluz — APPROVED** [review]
Nice yeah, much better
